### DISCUSSION TITLE
Subtle difference in default marker sequence for unordered lists in `reaveljs` and `html`

### DISCUSSION THREAD

**Discussion by juliantao** [quarto-dev/quarto-cli#1014] [Q&A] [answered]
The markers of unordered lists are rendered differently in `html` and `revaljs` format.

MREs:
```markdown
---
title: lists-html
format: html
---

# Lists

* Unordered list
  + sub-item 1
  + sub-item 2
    - sub-sub-item
```

```markdown
---
title: lists-revealjs
format: revealjs
---

# Lists in revealjs

* Unordered list
  + sub-item 1
  + sub-item 2
    - sub-sub-item
```

These are rendered to,
![Screenshot from 2022-05-28 14-18-06](https://user-images.githubusercontent.com/60116296/170843248-4ce698e3-268f-4095-bde0-23dc47c4a64c.png)
and
![Screenshot from 2022-05-28 14-19-08](https://user-images.githubusercontent.com/60116296/170843252-b9615b16-4bdd-47af-b535-e2aa8609c289.png)

It is a subtle difference and I am being picky here :wink: . The styles can be changed using `css`, but I would appreciate more consistency in the default styles. The `html` format rendering seems more comfortable to me.

**Comment by jjallaire** [comment, marked as answer]
I think slides and documents are different enough (and bullets distinctively important enough in slides) that I'd prefer to defer to the underlying themes for now (esp. b/c this is user configurable via CSS)

**Reply by juliantao** [reply]
Fair enough. Thank you for the quick reply!
